operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -1015,8 +1021,23 @@ private V1Probe createReadinessProbe(TuningParameters.PodTuning tuning) {
       boolean istioEnabled = getDomain().isIstioEnabled();
       if (istioEnabled) {
         int istioReadinessPort = getDomain().getIstioReadinessPort();
-        readinessProbe =
-            readinessProbe.httpGet(httpGetAction(READINESS_PATH, istioReadinessPort, false));
+        int localAdminPort = getLocalAdminProtocolChannelPort();
```

Comment:
The variable `localAdminPort` doesn't look to be used?

Reply:
fixed